### ISSUE TITLE
Fix weaken threads calculation

### ISSUE THREAD

**Issue by chrisrabe** [chrisrabe/bitburner-automation#8]
## Background

**Comment from colin bougie**

I noticed something while going through the code that I think is an error.  The number of weakenThreads should use the difference between the current security and the security threshold. Right now, it calculates the number of weaken threads necessary to go from 0 security to our threshold, which doesn't really make sense.

## Fix

- Update the weaken threads calculation for `launch-fleets` and the new `gimme-more-money` script.

**Comment by chrisrabe** [comment]
Updated thread calculations to:

```
		const weakenEffect = ns.weakenAnalyze(1);
		const secToDecrease = Math.abs(ns.getServerSecurityLevel(node) - secThresh);
		const weakenThreads = weakenEffect > 0 ? Math.round(secToDecrease / weakenEffect) : 0;
```

**Comment by chrisrabe** [comment]
Implemented in #19
